slp_mtmt/tests/integration/test_mtmt_processor.py

```diff
@@ -128,3 +141,95 @@ def test_run_valid_mappings(self):
+        assert otm.trustzones[0].representations is None
+
+        # AND we check the trust zone with representations
+        assert len(otm.trustzones[1].representations) == 1
+        assert otm.trustzones[1].representations[0].id == '24cdf4da-ac7f-4a35-bab0-29256d4169bf-representation'
+        assert otm.trustzones[1].representations[0].name == 'Private Secured Cloud Representation'
+        assert otm.trustzones[1].representations[0].representation == 'example-project-diagram'
+        assert otm.trustzones[1].representations[0].size == {'width': 371, 'height': 308}
+        assert otm.trustzones[1].representations[0].position == {'x': 744, 'y': 142}
+
+        # AND we check the component representations
+        assert len(otm.components) == 4
+        representation = otm.components[0].representations[0]
+        assert representation.id == '53245f54-0656-4ede-a393-357aeaa2e20f-representation'
+        assert representation.name == 'Accounting PostgreSQL Representation'
+        assert representation.representation == 'example-project-diagram'
+        assert representation.size == {'height': 100, 'width': 100}
+        assert representation.position == {'x': 231, 'y': 40}
+        assert otm.components[1].representations is None
+        representation = otm.components[2].representations[0]
+        assert representation.id == '5d15323e-3729-4694-87b1-181c90af5045-representation'
+        assert representation.name == 'Public API v2 Representation'
+        assert representation.representation == 'example-project-diagram'
+        assert representation.size == {'height': 100, 'width': 100}
+        assert representation.position == {'x': 21, 'y': 101}
+        representation = otm.components[3].representations[0]
+        assert representation.id == '91882aca-8249-49a7-96f0-164b68411b48-representation'
+        assert representation.name == 'Azure File Storage Representation'
+        assert representation.representation == 'example-project-diagram'
+        assert representation.size == {'height': 100, 'width': 100}
+        assert representation.position == {'x': 230, 'y': 169}
+
+    @mark.parametrize('source, expected', [
+        (MTMT_EXAMPLE_POSITION, OTM_EXAMPLE_POSITION),
+        (MTMT_EXAMPLE_1LINE, OTM_EXAMPLE_1LINE),
+        (MTMT_EXAMPLE_1ORPHAN, OTM_EXAMPLE_1ORPHAN)
+    ])
+    def test_coordinates_borders(self, source, expected):
+        # GIVEN a valid MTMT file
+        source_file = get_byte_data(source)
+
+        # AND a valid MTMT mapping file
+        mapping_file = get_byte_data(DEFAULT_MAPPING_FILE)
+
+        # AND the expected OTM
+        expected_otm = json.loads(get_byte_data(expected))
+
+        # WHEN the MTMT file is processed
+        otm = MTMTProcessor(SAMPLE_ID, SAMPLE_NAME, source_file, [mapping_file]).process()
+
+        # AND we get the json OTM
+        otm_json = otm.json()
+
+        # THEN we check the result is as expected
+        assert otm_json == expected_otm
```

Comment:
Should we use the `slp_base.tests.util.otm.validate_and_diff` utility method here for consistency with other tests?

Reply:
Are you sure make sense to validate again the otm when its validated previously?
Make sense to use a void regular expression when we don't need it?
Have you compared the simplicity of trace diffs between the currently code and the validate_and_diff ?
Imho in this case is cleaner the current way